docs/sources/mimir/manage/tools/mimirtool.md

```diff
@@ -218,6 +218,14 @@ The following command retrieves a single rule group and prints it to the termina
 mimirtool rules get <namespace> <rule_group_name>
 ```
 
+To save the rule group for editing and re-upload to mimir, simply add the flag `--save-file` and
+to desired output location which is defaulted to the current directory. The output file has the
+format required by `mimirtool rules load`.
+
+```bash
+mimirtool rules get <namespace> <rule_group_name> --save-file --output-dir="rules"
+```
+
```

Comment:
- Use semantic line breaks: 
[Semantic line breaks](https://sembr.org/) are better for future line based diffing in the GitHub UI and result in the same HTML output.
- Remove "simply", what is simple is subjective.
- Reword for clarity.

````suggestion
To save the rule group for editing and re-upload to Mimir, use the `--save-file` flag.
The output file has the format required by `mimirtool rules load`.

The default output directory is the current directory.
To output the file to a different directory, use the `--output-dir` option.

For example, to save the file in the `rules` subdirectory:

```bash
mimirtool rules get <namespace> <rule_group_name> --save-file --output-dir=rules
```

````

Reply:
Thanks for your suggestion. Also made me realize that the doc is out of date even within the scope of this PR since the redundant flag `--save-file` was removed earlier.